examples/worlds/apply_link_wrench.sdf

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" ?>
+<!--
+
+  Demostrates the ApplyLinkWrench plugin.
+
+  When simulation starts, the box model starts moving due to the persistent force defined in this file.
+
+  Wrenches can also be applied through transport.
+  For example, apply a force to the cylinder model (i.e. its canonical link) with:
+
+    ign topic -t "/world/apply_link_wrench/wrench" -m ignition.msgs.EntityWrench  -p "entity: {name: 'cylinder', type: MODEL}, wrench: {force: {y: 1000, z: 10000}}"
+
+  Apply a wrench to the cylinder link with:
+
+    ign topic -t "/world/apply_link_wrench/wrench" -m ignition.msgs.EntityWrench  -p "entity: {name: 'cylinder::link', type: LINK}, wrench: {force: {y: -1000, z: 10000}, torque: {x: -1000}}"
+
+  Clear the persistent force from the box with:
+
+    ign topic -t "/world/apply_link_wrench/wrench/clear" -m ignition.msgs.Entity  -p "name: 'box', type: MODEL"
+
+  Apply a persistent force to the cylinder with:
+
+    ign topic -t "/world/apply_link_wrench/wrench/persistent" -m ignition.msgs.EntityWrench  -p "entity: {name: 'cylinder', type: MODEL}, wrench: {force: {x: -20}}"
```

Comment:
nit: applying the persistent force command could be before clearing it

Reply:
The command above clears the persistent force on the box, which is set from the SDF though. Do you still think it would be better to reverse these 2 commands?

Reply:
Flipped them in [cc2bdb1](https://github.com/gazebosim/gz-sim/pull/1593/commits/cc2bdb1f1ebb37f379e0c4ffe04c077c27b30995)